### PR TITLE
add grad warning strings for experiments

### PR DESCRIPTION
**Pull request by johngruen**
We need to merge this after we tag for l10n, also the cliqz string is pending signoff from Javaun.

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3353/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/70d26346e10abc1f109fe0c78a9ded0b257b61d1/

**Review by clouserw — APPROVED** [review]

File: content-src/experiments/snooze-tabs.yaml [hunk 1]
```diff
@@ -96,4 +96,7 @@ contributors:
     title: 'Softvision QA'
     avatar: /static/images/experiments/avatars/ciprian.jpg
 created: '2017-02-22T17:00:00.00Z'
+graduation_url: http://example.com/graduation-report
```
Comment (pdehaan) [inline content-src/experiments/snooze-tabs.yaml:99]: Do we want the fake `graduation_url` here? (it isn't in **cliqz**, **min vid**, or **voice fill**)

Comment (johngruen) [reply content-src/experiments/snooze-tabs.yaml:99]: @pdehaan doesn't block...we'll have to do a separate PR either way

**Comment by SoftVision-PaulOiegas** [comment]
Find the experiments graduation decisions by spying the PR code. 😆 That's one way QA can find out what will happen with each experiment. 

giphy

File: content-src/experiments/min-vid.yaml [hunk 2]
```diff
@@ -98,4 +98,6 @@ contributors:
     title: 'Engineering Contributor'
     avatar: /static/images/experiments/min-vid/avatars/noitidart.jpg
 created: '2016-09-22T00:07:28.847430Z'
-order: 4
\ No newline at end of file
+order: 4
+eol_warning: Min Vid is leaving Test Pilot soon, and unfortunately, we will be disabling it for all users.
```
Comment (chuckharmston) [inline content-src/experiments/min-vid.yaml]: I don't think the second comma here is necessary.

**Review by chuckharmston — CHANGES_REQUESTED** [review]
Small grammar nit.

**Comment by chuckharmston** [comment]
Ping @flodolo for review on these substantial string changes.

**Review by flodolo — APPROVED** [review]
No issues on my side

**Review by chuckharmston — APPROVED** [review]

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3353/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/d5656c05ae5664798a22a8e2f115396636499869/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3353/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/180e2b17332ab73a4e2c147111b6b5298efc6410/

**Pull request merged**